libpod/container_internal.go

```diff
@@ -271,6 +272,43 @@ func (c *Container) export(path string) error {
 	return err
 }
 
+func (c *Container) addImageVolumes() error {
+	mountPoint := c.state.Mountpoint
+	if !c.state.Mounted {
```

Comment:
This should not happen here, use c.mountStorage() if you need storage mounted.

Reply:
I think it would be best if we ran this in Init() when we generated the spec and passed in the generator, g, so we can modify the created spec. The container will already be mounted by then, so this mount code can be removed and replaced with `if !c.state.Mounted { return errors.Wrapf(ErrInternal, "container is not mounted") }` or similar

Reply:
fixed